frame/society/src/lib.rs

```diff
@@ -0,0 +1,1372 @@
+
+			let vote = if approve { Vote::Approve } else { Vote::Reject };
+			<Votes<T, I>>::insert(&candidate, &voter, vote);
+
+			Self::deposit_event(RawEvent::Vote(candidate, voter, approve));
+		}
+
+		/// As a member, vote on the defender.
+		///
+		/// The dispatch origin for this call must be _Signed_.
+		///
+		/// Parameters:
+		///
+		/// - `approve`: A boolean which says if the candidate should be
+		///              approved (`true`) or rejected (`false`).
+		///
+		/// # <weight>
+		/// - One storage read and O(log M) search to check user is a member.
+		/// - One storage write to add vote to votes.
+		/// - One event.
+		/// # </weight>
+		///
+
+		#[weight = SimpleDispatchInfo::FixedNormal(20_000)]
+		pub fn defender_vote(origin, approve: bool) {
+			let voter = ensure_signed(origin)?;
+			ensure!(Self::is_member(&voter), Error::<T, I>::NotMember);
+
+			let vote = if approve { Vote::Approve } else { Vote::Reject };
+			<DefenderVotes<T, I>>::insert(&voter, vote);
+
+			Self::deposit_event(RawEvent::DefenderVote(voter, approve));
+		}
+
+		/// Transfer the first matured payout for the sender and remove it from the records.
+		///
+		/// This extrinsic can be called multiple times to claim all matured payouts.
+		///
+		/// The dispatch origin for this call must be _Signed_.
+		///
+		/// # <weight>
+		/// - One storage read and O(log M) search to check signer is a member.
+		/// - One storage read to get all payouts for a member.
+		/// - One storage read to get the current block number.
+		/// - Up to one currency transfer call.
+		/// - One storage write or removal to update the member's payouts.
+		/// # </weight>
+		///
+
+		#[weight = SimpleDispatchInfo::FixedNormal(30_000)]
+		pub fn payout(origin) {
+			let who = ensure_signed(origin)?;
+
+			ensure!(Self::is_member(&who), Error::<T, I>::NotMember);
+
+			let mut payouts = <Payouts<T, I>>::get(&who);
+			if let Some((when, amount)) = payouts.first() {
+				if when <= &<system::Module<T>>::block_number() {
+					T::Currency::transfer(&Self::payouts(), &who, *amount, KeepAlive)?;
+					payouts.remove(0);
+					if payouts.is_empty() {
+						<Payouts<T, I>>::remove(&who);
+					} else {
+						<Payouts<T, I>>::insert(&who, payouts);
+					}
+					return Ok(())
+				}
+			}
+			Err(Error::<T, I>::NoPayout)?
+		}
+
+		/// Found the society.
+		///
+		/// This is done as a discrete action in order to allow for the
+		/// module to be included into a running chain.
+		///
+		/// The dispatch origin for this call must be from the _FounderOrigin_.
+		///
+		/// Parameters:
+		///
+		/// - `founder` - The first member and head of the newly founded society.
+		///
+		/// # <weight>
+		/// - One storage read to check `Head`.
+		/// - One storage write to add the first member to society.
+		/// - One storage write to add new Head.
+		/// - One event.
+		///
+		/// This is marked as `Operational` since it is a low level system action.
```

Comment:
Incorrect. It's nowhere near "low-level" enough to be operational. Operational stuff is for extrinsics that are critical to the security of the network.

Reply:
Ack. Misunderstood what operational extrinsics should be used for.